spec/linter-rubocop-spec.js

```diff
@@ -136,6 +138,29 @@ describe('The RuboCop provider for Linter', () => {
     );
   });
 
+  describe('respects .ruby-version when .rubycop.yml has not defined ruby version', () => {
+    let rubocopYmlText;
+
+    beforeEach(() => {
+      rubocopYmlText = readFileSync(yml, 'utf8');
```

Comment:
Why are you reading this in every time? Preferably you would have two folders, one with the file and one without so you can verify that it is actually making a difference. Barring that you should just be writing and deleting the file. This current setup of "read in, blank out, write back" is just causing extra I/O.